test/test_helper.js

```diff
@@ -0,0 +1,4 @@
+import chai from 'chai';
```

Comment:
not familiar at all with chai, but i guess i can cross that bridge when i come to it